dependencies/secret/empty_secret_service.go

```diff
@@ -0,0 +1,14 @@
+package secret
+
+import (
+	"context"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+)
+
+func (ess EmptySecretService) LoadSecret(ctx context.Context, k string) (string, error) {
+	return "", errors.Newf(codes.NotFound, "secret key %q not found", k)
+}
+
+type EmptySecretService struct {
```

Comment:
Can you add a comment here explaining how the service behaves? i.e. it errors loading any secret.

Reply:
@nathanielc do we need to write testing for updates like this? It didn't look like the previous version `environment_secret_service` had any tests. Just on `secret_service`

Reply:
We could, but the test doesn't get us much as all it does is validate the method returns an error. Behavior which we can clearly audit in the code. I would not be opposed to adding a simple unit test for the service, but I wouldn't require it either. I'll leave it up to the author in this case.